lib/utils/deep-sort-object.js

```diff
@@ -3,7 +3,9 @@ var sortedObject = require('sorted-object')
 
 module.exports = function deepSortObject (obj, sortBy) {
   if (obj == null || typeof obj !== 'object') return obj
-  if (obj instanceof Array) return obj.sort(sortBy)
+  if (obj instanceof Array) return obj.map(function (x) {
```

Comment:
This originally came in with https://github.com/npm/npm/commit/2f3c83a59f7387cff41ee0894707d1cffed04825 to resolve #5779 by making the npm-shrinkwrap.json output consistent.

npm-shrinkwrap.json no longer includes the entire package manifest, so I'm not sure how useful it is anymore.